opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -172,9 +172,118 @@ message MetricDescriptor {
   }
   Type type = 4;
 
+  // Temporality is the temporal quality values of a metric have. It
+  // describes how those values relate to the time interval over which they
+  // are reported.
+  enum Temporality {
+    // TEMPORALITY_INVALID is the default Temporality, it MUST not be
+    // used.
+    TEMPORALITY_INVALID = 0;
+
+    // INSTANTANEOUS is a metric whose values are measured at a particular
+    // instant. The values are not aggregated over any time interval and are
+    // unique per timestamp.
+    INSTANTANEOUS = 1;
```

Comment:
Do we want to specify that a start time is not expected for this?

Reply:
I plan to have comment for time to be kept with the PR to update time: https://github.com/open-telemetry/opentelemetry-proto/pull/137/files#diff-ad849336b0a70e8a18346421908f69edR98-R100

Reply:
Ehh, on second thought, couldn't hurt.